annotation_full.wdl

```diff
@@ -245,6 +264,50 @@ task stage {
    }
 }
 
+task make_map_file {
+
+    String proj_id
+    String  prefix=sub(proj_id, ":", "_")
+    File input_file
+    String container
+    String output_file = "${prefix}_map.fasta"
+    Int min_seq_length = 150      # default value
+    Int unknown_gap_length = 100  # default value
+
+
+  command <<<
+  find_prefix=`grep ${proj_id} ${input_file} | head -1`
```

Comment:
Why can't we just get this info from the input json proj variable?

Reply:
checks for whether the prefix is there or not to decrease the number of variables that need human input.